packages/shopping-cart/src/pages/Login.jsx

```diff
@@ -0,0 +1,22 @@
+import {Link} from 'react-router-dom';
+export default function Login(props) {
+  return(
+    <div className="bg-light p-5 my-5 col-5 mx-auto border border-2">
+      <h3 className="text-center mb-4">Enter your credentials</h3>
+      <form className="">
+        <div className="mb-3">
```

Comment:
extract similar items into reusable components such as inputs , labels for forms